Client/Frontend/Fakespot/FakespotViewModel.swift

```diff
@@ -5,5 +5,63 @@
 import Foundation
 
 class FakespotViewModel {
-    // Define your properties and methods for the ViewModel here
+    enum ViewState<T> {
+        case loading
+        case loaded(T)
+        case error(Error)
+    }
+
+    @Published private(set) var state: ViewState<ProductAnalysisData?> = .loading
```

Comment:
I made the state of the screen be Published so we can observe it. Of course there are other ways of doing it if the teams feels like this is not a pattern we want to use.